lib/ansible/cli/galaxy.py

```diff
@@ -12,6 +12,7 @@
 import textwrap
 import time
 import yaml
+import json
```

Comment:
This is injected right between two `yaml` imports which is also a third-party. Please move it up, before line 8 (all other imports are sorted alphabetically).